_includes/header.html

```diff
@@ -1,35 +1,33 @@
-<header class="site-header" role="banner">
-  <!--<img src="C:\Source_Code\sunair-in.github.io\logo.jpeg" width="100px" 
-  height="100px">-->
-  <img src="/assets/images/logo.jpeg" width="100px" height="100px" />
-
+<header class="site-header">
+  
   <div class="wrapper">
-    {%- assign default_paths = site.pages | map: "path" -%} {%- assign
-    page_paths = site.header_pages | default: default_paths -%} <a
-    class="site-title" rel="author" href=" {{ relative_url }} > {{ site.title |
-    escape }} /a> {%- if page_paths -%}
```

Comment:
anchor tag was closed properly. Note the `/a>`. This was fixed by restoring the whole header.html from minima source.